### PR TITLE
Update Sessions::compareTo to sort in reverse chronological order

### LINKED ISSUE #332: Bug: Session is sorted in chronological instead of reverse chronological order

### PR DESCRIPTION
**Pull request by Berted**
Resolves #332.

### CODE REVIEW

File: src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java [hunk 1]
```diff
@@ -25,7 +25,7 @@ public class ModuleClass implements Identity<ModuleClass>, Comparable<ModuleClas
 
     private final String className;
 
-    private final List<Session> sessions;
+    private final ArrayList<Session> sessions;
```
Comment (Bubbl3T) [inline src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java]: Perhaps we should use `List` instead of `ArrayList` for better abstraction?

**Review by Bubbl3T — CHANGES_REQUESTED** [review]
LGTM!

File: src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java (hunk 1 above)
Comment (Berted) [reply src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java]: I think it's okay for it to be defined as `ArrayList` internally, as previously accepting just `List` allowed a `List` from another class to be passed instead, which may lead to undesirable behaviour.

Comment (RezwanArefin01) [inline src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java]: Should this be a `UniqueList` instead?

**Review by RezwanArefin01 — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java (hunk 1 above)
Comment (RezwanArefin01) [reply src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java]: Should be resolved via #342. Maybe it's a good idea to wait for that PR to merge first.

**Review by RezwanArefin01 — APPROVED** [review]
Looks good to merge!

**Review by Bubbl3T — APPROVED** [review]
LGTM!

**Pull request merged**